plugins/linux/vm.sh

```diff
@@ -4,18 +4,18 @@ print_vm() {
     printf "%s\`%s\tL\t%s\n" $1 $2 $3
 }
 
-MEM=($(free -b | grep ^Mem:))
+MEM=($(free -k | grep ^Mem:))
 MEM_TOTAL=${MEM[1]}
 # For consistency across platforms, count cache as free, not used
 let MEM_USED=${MEM[1]}-${MEM[3]}-${MEM[5]}-${MEM[6]}
 let MEM_FREE=${MEM[3]}+${MEM[5]}+${MEM[6]}
-MEM_PERC=`echo "scale=2; $MEM_USED/$MEM_FREE" | bc`
+let MEM_PERC=$MEM_USED/$MEM_TOTAL
```

Comment:
When I test this on my Linux box, MEM_PERC is alway 0 because bash can't handle non-integer math.

I'm open to a bash-only solution, but that's why we used bc.